ironfish/src/rpc/routes/chain/snapshotChain.test.ts

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { Assert } from '../../../assert'
+import { getBlockSize, writeBlock } from '../../../network/utils/block'
+import { useMinerBlockFixture } from '../../../testUtilities'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+import { SnapshotChainStreamResponse } from './snapshotChain'
+
+describe('Route chain/snapshotChainStream', () => {
+  const routeTest = createRouteTest()
+
+  describe('with no maxBlocksPerChunk parameter', () => {
+    it('correctly returns a serialized list of blocks', async () => {
+      const { chain, strategy } = routeTest
+      await chain.open()
+      strategy.disableMiningReward()
+
+      const genesis = await chain.getBlock(chain.genesis)
+      Assert.isNotNull(genesis)
+
+      const blockA1 = await useMinerBlockFixture(chain, 2)
+      await expect(chain).toAddBlock(blockA1)
```

Comment:
Do we need these expects here? Seems like this is just testing the fixture helpers

Reply:
Didn't realize `expect(chain).toAddBlock(blockA1)` is actually adding the block so that one makes sense

Reply:
Yes, you're correct. Additionally, we do this in a fair number of other test suites as well, so I kept this in line with the others.